5-infrastructure/modules/gke/outputs.tf

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "name" {
+  description = "Cluster name"
+  value       = module.private-cluster.name
+}
+
+output "type" {
+  description = "Cluster type (regional / zonal)"
+  value       = module.private-cluster.type
+}
+
+output "location" {
+  description = "Cluster location (region if regional cluster, zone if zonal cluster)"
+  value       = module.private-cluster.location
+}
+
+output "region" {
+  description = "Cluster region"
+  value       = module.private-cluster.region
+}
+
+output "zones" {
+  description = "List of zones in which the cluster resides"
+  value       = module.private-cluster.zones
+}
+
+output "endpoint" {
+  sensitive   = true
+  description = "Cluster endpoint"
+  value       = module.private-cluster.endpoint
+}
+
+output "min_master_version" {
```

Comment:
Do we need this? Only output values which will be used my other modules or by the user here and throughout